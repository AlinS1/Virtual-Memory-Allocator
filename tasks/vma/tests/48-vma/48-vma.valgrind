==3002== Memcheck, a memory error detector
==3002== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==3002== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==3002== Command: /mnt/c/Users/alins/OneDrive/Desktop/SDA_tema1/vma
==3002== Parent PID: 3001
==3002== 
==3002== 
==3002== HEAP SUMMARY:
==3002==     in use at exit: 6 bytes in 1 blocks
==3002==   total heap usage: 40 allocs, 39 frees, 1,974 bytes allocated
==3002== 
==3002== 6 bytes in 1 blocks are still reachable in loss record 1 of 1
==3002==    at 0x483B7F3: malloc (in /usr/lib/x86_64-linux-gnu/valgrind/vgpreload_memcheck-amd64-linux.so)
==3002==    by 0x10A6E1: create_string (vma.c:396)
==3002==    by 0x1096DB: main (main.c:66)
==3002== 
==3002== LEAK SUMMARY:
==3002==    definitely lost: 0 bytes in 0 blocks
==3002==    indirectly lost: 0 bytes in 0 blocks
==3002==      possibly lost: 0 bytes in 0 blocks
==3002==    still reachable: 6 bytes in 1 blocks
==3002==         suppressed: 0 bytes in 0 blocks
==3002== 
==3002== For lists of detected and suppressed errors, rerun with: -s
==3002== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
