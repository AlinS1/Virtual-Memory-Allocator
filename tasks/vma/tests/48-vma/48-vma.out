gcc -g -o vma main.c vma.c -Wall -Wextra -std=c99 -I.
